src/handlers/dispersion.ts

```diff
@@ -82,9 +83,31 @@ export const getTrendingAddresses = async (): Promise<
   }
 };
 
-export const getAdditionalAssetData = async (address: EthereumAddress) => {
+export const getAdditionalAssetData = async (
+  address: EthereumAddress,
+  network: Network | string
+) => {
+  let chainId: number;
+
+  switch (network) {
+    case Network.arbitrum:
+      chainId = 42161;
+      break;
+    case Network.optimism:
+      chainId = 10;
+      break;
+    case Network.polygon:
+      chainId = 137;
+      break;
+    default:
+      chainId = 1;
+      break;
+  }
```

Comment:
https://github.com/rainbow-me/rainbow/blob/develop/src/utils/ethereumUtils.ts#L320

We've got a util for this.

Reply:
Thanks, fixed!